pyrasterframes/src/main/python/pyrasterframes/__init__.py

```diff
@@ -56,6 +56,39 @@ def _kryo_init(builder):
     return builder
 
 
+def get_spark_session():
```

Comment:
This is amazing!

I wonder if there there is some way to get the `pyspark` shell to call this when creating a session? Jupyter?

Reply:
in jupyter it should be easy to set up the kernel to run a small script when it initializes.  Less sure about doing that automatically.

Reply:
@vpipkt  After thinking about it more, I think this function needs to be renamed... it's too generic and will get added to the global namespace with a `from pyrasterframes import *`.

My preference would be to add an extension on `SparkSession.Builder`, like `withKryoSerialization`, that does something like `.addRasterFramesJars`, which in turn does all this work and then passes it to `.config('spark.jars', ...)`. **However**, thanks to @jbouffard 's comment, its apparent we need to make sure we can do that while appending to any existing value of  `spark.jars`. If Spark doesn't make that possible then I think we can suffice with a less comprehensive `get_rasterframes_jars`.

Reply:
As you'll see, I did some partial work around this, putting common code associated with jar resolution in `utils.py`.

Reply:
As you'll see, I did some partial work around this, putting common code associated with jar resolution in utils.py.

Reply:
I like the idea of `utils.py` with some functions for the session builder and configs.

I would like to put a priority on making it easy for a new user to get going, as shown in proposed readme.  As far as the function naming, let's not worry too much about user doing `from pyrasterframes import *`;  short of overloading a python built in or `spark.sql.function`. 

The current state of naming seems very good to me.

See the following good references [1](https://medium.com/@s16h/importing-star-in-python-88fe9e8bd4d2) and [2](http://xion.io/post/code/python-all-wild-imports.html), regarding wildcard import and `__all__`.

Reply:
Really good to know... I didn't realize `__all__` was as limited in purpose.